### DISCUSSION TITLE
Kris Rasmussen - Figma

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#345] [Announcements]
- https://www.linkedin.com/in/kristopherrasmussen/
- https://x.com/kris_rasmussen
- https://github.com/krisr

## Questions

- [x] For those who might have been living under a rock: What is Figma? How did you get involved?

### Figma: The Tech

- [x] What "bleeding edge" browser APIs were required for something like figma to work? What new browser APIs have you excited?
- [x] Figma famously uses WASM to function. What parts are WASM? What core experiences wouldn't be possible without it?
- [x] What are some of the tough challenges of deploying/running Figma?

#### Dev Mode

- [x] What is dev mode? How does it make dev handoff simpler? What are some of the key understandings that led to the experience?
- [x] Dev mode has been out for awhile now. How has it gone? 
- [x] It seems like a variety of cool projects came out of "dev mode'". What are some of your favorites?

#### AI

- [x] What is figma doing with AI? Many companies just slap a chatbot somewhere, how is figma shipping things that are actually useful?
- [x] Does figma do the AI in the browser or it it utilizing the cloud?
- [x] The holy grail to me seems to be "no handoff" Aka the tool ends up making the markup. Has figma ever experimented with this? Might LLMs get us closer to this dream?
- [x] Do you think AI will change what it means to be a designer? What it means to be an engineer?

### Future

- [ ] What do you think is still left to be solves in product development via figma?
- [x] What do you think the future of building UIs looks like? Will our tools merge? Will AI make parts of the problem go away?

**Comment by just-be-dev** [comment]
https://github.com/figma/code-connect
